### ISSUE TITLE
Voiceover is not announcing the current count when we double tap on "click me" button.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#6216]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yMAS; #A11yTCS; #Benchmark; #iOS; #iPad; #WCAG1.3.1; #GH_.NETMAUI_IOS_Mar2022; #Info and Relationships​; #AINotInScope;

### Description

Voiceover is not announcing the current count when we double tap on "click me" button.

### Steps to Reproduce

1.Install and open ".NET MAUI Template" app.
2..Net MAUI template screen will open.
3.Swipe till "Click me" button.
4.Verify Voiceover is announcing the current count when we double tap on "click me" button or not.

https://user-images.githubusercontent.com/100186395/163923977-cfd1bfdd-2bd7-440c-a1ca-3f2d9023ef80.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4.1(iPad)

### Relevant log output

```shell
Voiceover should announce the current count when we double tap on "click me" button i.e. when double tap on "click me" button voiceover should read "Current count Number".
```

**Comment by v-longmin** [comment]
Verified repro on VS 17.3.0 Preview 1.0 [32418.321.main]. Repro with maui sample template project.

**Comment by rachelkang** [comment]
This should be fixed in our latest version!

**Comment by kristinx0211** [comment]
verified on windows and android with 17.3.0 Preview 2.0 [32507.29.main], not repro. close it. feel free to open new issue if you still hit this failure with same/newer environment.

**Comment by Karan4495** [comment]
Verify issue in ".NET MAUI Template" app. issue is fixed so closing the bug
IOS Version: iPad(15.4.1)
added attachment for references

https://user-images.githubusercontent.com/100186395/167605472-a3da641f-68ee-4102-a7f9-f2f911eee3a9.mp4
